website/docs/tutorials/app-demo-tutorial.md

```diff
@@ -0,0 +1,566 @@
+---
+title: Demo Application
+---
+<!--
+Author(s): Mark Fulton
+Reviewer(s): Thomas, Mike Vitteglio, Sebastian Bury
+
+Resources (remove before publication)
+
+- [Diataxis/](https://diataxis.fr/tutorials/)
+- [Unleash Docs GH](https://github.com/Unleash/unleash/tree/main/website/docs)
+- [Unleash QSG](https://github.com/Unleash/unleash/blob/main/website/docs/tutorials/quickstart.md?plain=1)
+-->
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+# Introduction
+
+This tutorial will guide you through the process of setting up a demo web application with working feature toggles provided by an Unleash instance.  
+In a short period of time, it will demonstrate the power of Unleash as a runtime control mechanism to control the state of software while not being bound by release and deploy processes.  
+
+Specifically, you will learn:
+
+- *Using Next.js, how to incorporate Unleash client (frontend) and server-side (backend) SDKs into the application source code*
```

Comment:
I don't think there's a right or wrong answer for this one, but here's how I justify my usage of the terms.

In previous roles we standardized on the [Google developer doc style guide](https://developers.google.com/style/word-list#letter-b) for our docs and training materials, which mention:

> backend; not back-end or back end

So they are taking a simplistic approach by only leveraging `backend` and not the other variants.

Linguistically a commonly used dictionary [references all three variants](https://www.merriam-webster.com/dictionary/back%20end) - so again, both approaches are valid. The single word approach that Google takes may be simpler to enforce in a style guide, however.

Reply:
Ah, that's an interesting point. And yeah, I don't think there is a clear right answer here. I agree that writing it as a single word always would make it easier to encode in a style guide (not to mention it'd be easier to grok). I also see a lot of it in use internally. 

I'd like to try and summon @JarrodSC to hear if he's got any thoughts on the matter? This might be something we want to standardize on across docs, blogs, etc?